molecularnodes/ui/panel.py

```diff
@@ -17,13 +17,13 @@
 bpy.types.Scene.MN_panel_import = bpy.props.EnumProperty(
     name="Method",
     items=(
-        ('pdb', "PDB", "Download from the PDB", 0),
-        ('local', "Local", "Open a local file", 1),
-        ('md', "MD", "Import a molecular dynamics trajectory", 2),
-        ('density', "Density", "Import an EM Density Map", 3),
-        ('star', 'Starfile', "Import a .starfile mapback file", 4),
+        ('pdb', "PDB", "Download from the PDB.", 0),
```

Comment:
Blender (_verty annoyingly_) auto-adds a period to the ends of tooltips. If you hover over the tooltips when they are defined like this, then it will end with a double-period. You can see that even I am inconsistent with them - so if you could remove them all that would be great.

Reply:
That is very annoying. I'll go through and remove the trailing periods.